rp235x-hal/src/timer.rs

```diff
@@ -6,7 +6,7 @@
 //!
 //! Each of the 4 alarms can match on the lower 32 bits of Counter and trigger an interrupt.
 //!
-//! See [Chapter 4 Section 6](https://datasheets.raspberrypi.org/rp235x/rp235x_datasheet.pdf) of the datasheet for more details.
+//! See [Section 12.8](https://datasheets.raspberrypi.org/.pdf) of the datasheet for more details.
```

Comment:
```suggestion
//! See [Section 12.8](https://rptl.io/rp2350-datasheet) of the datasheet for more details.
```

Reply:
How was it even possible I did this? I'm baffled.